ui/imports/shared/popups/SendModal.qml

```diff
@@ -148,15 +151,151 @@ StatusModal {
                 contentHeight: addressSelector.height + networkSelector.height + gasSelector.height + gasValidator.height
                 clip: true
 
+                ColumnLayout {
+                    id: assetAndAmmountSelector
+                    anchors.top: parent.top
+                    anchors.right: parent.right
+                    anchors.left: parent.left
+                    RowLayout {
+                        spacing: 16
+                        StatusBaseText {
+                            //% "Send"
+                            text: qsTrId("command-button-send")
```

Comment:
`qsTr("Send")`

Reply:
this is already added as a translation can be reused here?